### PR TITLE
Align active maintainers with reality

### PR DESCRIPTION
**Pull request by jimirocks**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/766/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/628/ : FAILURE in 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/889/ : SUCCESS in 33s

File: README.md [hunk 1]
```diff
@@ -59,8 +59,7 @@ Pavel Kolesnikov [ <mailto:pavel@gooddata.com> / [@koles](http://twitter.com/kol
 
 **Actively developed and maintained by**
 
-- [Tomas Korcak](https://github.com/korczis) [ <mailto:korczis@gmail.com> / [@korczis](http://twitter.com/korczis) ]
-- [Tomas Svarovsky](https://github.com/fluke777) [ <mailto:svarovsky.tomas@gmail.com> / [@fluke777](http://twitter.com/fluke777) ]
```
Comment (kubamahnert) [inline README.md]: maybe move Korczis and Svara into contributors instead of getting rid of them completely?

File: README.md [hunk 2]
```diff
@@ -59,8 +59,7 @@ Pavel Kolesnikov [ <mailto:pavel@gooddata.com> / [@koles](http://twitter.com/kol
...
+- [Jan Zdráhal](https://github.com/panjan) [ <mailto:jan.zdrahal@gooddata.com> ]
```
Comment (panjan) [inline README.md]: We should add @kubamahnert and @Hahihula too.

File: README.md (hunk 1 above)
Comment (jimirocks) [reply README.md]: well their credits are on contribution page and they are not contributors any more, may be we can move them to special thanks? or create new section - wht would be the name?

File: README.md (hunk 2 above)
Comment (jimirocks) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (jimirocks) [reply README.md]: ok I decided to move them just next to other contributors

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/769/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/631/ : SUCCESS in 55s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/892/ : SUCCESS in 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/776/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/652/ : SUCCESS in 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/913/ : SUCCESS in 39s

**Comment by kubamahnert** [comment]
Looks good!

**Pull request merged**